### PR TITLE
Issue #3324781: Add split text functionality to CKEditor 5

### PR DESCRIPTION
**Pull request by zibellino**
Implementation of the split text (split paragraph) functionality for CKEditor5.
Based on the [ckeditor5_plugin_starter_template](https://git.drupalcode.org/project/ckeditor5_dev/-/tree/1.0.x/ckeditor5_plugin_starter_template), and as such has the same dependencies.

Uses a custom split function instead of [SplitOperation](https://ckeditor.com/docs/ckeditor5/latest/api/module_engine_model_operation_splitoperation-SplitOperation.html), and the "Add in-between" button instead of it's own modal.

https://www.drupal.org/project/paragraphs_features/issues/3400542

### CODE REVIEW

File: paragraphs_features.ckeditor5.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  drupal:
+    label: Split paragraph
+    library: paragraphs_features/splitParagraph
+    admin_library: paragraphs_features/splitParagraph.admin
+    toolbar_items:
+      splitParagraph:
+        label: Split paragraph
+    elements:
+      - <p>
+      - <div>
```
Comment (milkovsky) [inline paragraphs_features.ckeditor5.yml]: missing NL

File: paragraphs_features.libraries.yml [hunk 2]
```diff
@@ -33,3 +33,14 @@ scroll_to_element:
...
+splitParagraph:
+  js:
+    js/build/split_paragraph.js: { preprocess: false, minified: true }
+  dependencies:
+    - core/ckeditor5
+
+splitParagraph.admin:
+  css:
+    theme:
+      css/split_paragraph.admin.css: {}
```
Comment (milkovsky) [inline paragraphs_features.libraries.yml]: missing NL

**Review by milkovsky — CHANGES_REQUESTED** [review]
The code both looks and works good! There are 2 minor comments.

**Comment by milkovsky** [comment]
Also please create a d.o. issue for this feature.

**Review by milkovsky — APPROVED** [review]
looks and works good!

**Review by milkovsky — APPROVED** [review]
well done

**Review by milkovsky — APPROVED** [review]

**Review by milkovsky — APPROVED** [review]

**Comment by ol0lll** [comment]
Closing in favour of #80

**Pull request closed without merging**